### PR TITLE
New landing pages

### PR DESCRIPTION
**Pull request by sebbacon**

### CODE REVIEW

**Comment by sebbacon** [comment]
Bugs noticed:

* [] "Fetching data" spinner never disappears on http://staging.openprescribing.net/ccg/10Y/measures/
* [] Add some intro text to [alert email sample](http://staging.openprescribing.net/alert_example/)
* [] Form fields not appearing on [measures page](http://staging.openprescribing.net/ccg/10Y/measures/)
* [] Check only signing up for newsletter doesn't also sign up for alerts (when testing I did get "Thanks, you're now subscribed to monthly alerts" message - pretty sure didn't get a bookmark though)
* [] Write a test that includes the PPU total

**Comment by alexwalkercebm** [comment]
Graphs for practice landing pages don't appear, e.g. http://staging.openprescribing.net/practice/Y02847/
Looks fine for CCGs

**Comment by richiecroker** [comment]
Graphs not showing on practice landing page?
image

**Comment by richiecroker** [comment]
Otherwise - I like!

**Comment by HelenCEBM** [comment]
- [ ] On CCG measures pages need to make the map more zoomed out, as it is on the new dashboard page:
image
The more zoomed in one is not informative: 
image

**Comment by sebbacon** [comment]
There are discrepancies in PPU savings - this example is 09X:

image

versus:

image

File: openprescribing/frontend/tests/test_api_utils.py [hunk 1]
```diff
@@ -12,3 +12,12 @@ def do_long_running_query():
                 cursor = connection.cursor()
                 cursor.execute("select pg_sleep(0.01);")
         self.assertRaises(OperationalError, do_long_running_query)
+
+    def test_param_to_list(self):
+        from api.view_utils import param_to_list
+
+        self.assertEquals(param_to_list('foo'), ['foo'])
+        self.assertEquals(param_to_list('foo,bar'), ['foo', 'bar'])
+        self.assertEquals(param_to_list('foo,bar'), ['foo', 'bar'])
```
Comment (evansd) [inline openprescribing/frontend/tests/test_api_utils.py]: This test is identical to the one above. Was it supposed to test something else, or is an accidental dupe?

File: openprescribing/api/views_spending.py [hunk 2]
```diff
@@ -208,72 +208,28 @@ def price_per_unit(request, format=None):
...
 
-    sql = sql.format(
-        ppusavings_table=PPUSaving._meta.db_table,
-        practice_table=Practice._meta.db_table,
-        presentation_table=Presentation._meta.db_table,
-        dmdproduct_table=DMDProduct._meta.db_table,
-        dmdvmpp_table=DMDVmpp._meta.db_table,
-        ncsoconcession_table=NCSOConcession._meta.db_table,
-    )
+    sql = ppu_sql(extra_conditions=extra_conditions)
```
Comment (evansd) [inline openprescribing/api/views_spending.py]: This should be:
```python
    sql = ppu_sql(conditions=extra_conditions)
```

**Comment by HelenCEBM** [comment]
Is it possible to zoom in a bit on the maps? They are so zoomed out it's difficult to see which area is highlighted - I think it was clearer when we had the big marker as in previous version. Don't get me wrong, the area maps are better overall, just harder to see when zoomed out so far. After double clicking on the map once or twice it's better. 

On a related note, with the circular crop of the map, the controls aren't visible so it's not possible to zoom out again!

**Comment by HelenCEBM** [comment]
It would probably be useful to add a "Looking for something else?" note at the bottom of the dashboards to link to the other features such as the Analyse and Trends pages.

**Pull request merged**